container/docker/factory.go

```diff
@@ -49,6 +50,12 @@ var ArgDockerCA = flag.String("docker-tls-ca", "ca.pem", "path to trusted CA")
 // The namespace under which Docker aliases are unique.
 const DockerNamespace = "docker"
 
+// The retry times for getting docker root dir
+const rootDirRetries = 5
+
+//The retry period for getting docker root dir, Millisecond
+const rootDirRetryPeriod = 1000
```

Comment:
Please make the constant be a time.Duration, rather than an integer.  That way we don't have to worry about units.  I.E. `rootDirRetryPeriod = time.Second`

Reply:
Ok~I have corrected it.